std_msgs/Float32 thrust
std_msgs/Float32 roll_rate
std_msgs/Float32 pitch_rate
std_msgs/Float32 yaw_rate
